Setting paths to search for dnynmic libraries isn't very useful in Nix.

--- findlib-1.2.6/findlib.conf.in	2010-08-17 14:50:28.000000000 -0400
+++ findlib-1.2.6/findlib.conf.in	2010-11-24 09:46:55.179648354 -0500
@@ -1,2 +1,3 @@
 destdir="@SITELIB@"
 path="@SITELIB@"
+ldconf="ignore"
